fobi_custom/plugins/form_elements/fields/intercept/age_intercept/forms.py

```diff
@@ -12,7 +12,6 @@ class AgeInterceptForm(forms.Form, BaseFormFieldPluginForm):
     plugin_data_fields = [
         ("label", "How old are you?"),
         ("name", "name"),
-        ("age", ""),
```

Comment:
This wasn't hurting anything but is a stray copy-paste mistake that I've deleted from a couple plugins.